### PR TITLE
LPS-106854 - As a search developer, I want to use ADTs to customize the User Facet Widget

### PR DESCRIPTION
**Pull request by thektan**
http://issues.liferay.com/browse/LPS-106854

I made these changes ontop of the Folder Facet ADT changes, so I'll have to rebase again once it's in upstream.

### CODE REVIEW

**Comment by thektan** [comment]
ci:test:sf

**Comment by thektan** [comment]
ci:test:search

**Comment by thektan** [comment]
Pushed an additional commit since I realized there was a place I needed to resort.

File: modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/user/facet/template/UserFacetPortletDisplayTemplateHandler.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+			templateVariableGroups.get("fields");
+
+		templateVariableGroup.empty();
+
+		templateVariableGroup.addVariable(
+			"term-frequency", Integer.class,
+			PortletDisplayTemplateConstants.ENTRY, "getFrequency()");
+		templateVariableGroup.addVariable(
+			"term-name", String.class, PortletDisplayTemplateConstants.ENTRY,
+			"getDisplayName()");
```
Comment (oliv-yu) [inline modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/user/facet/template/UserFacetPortletDisplayTemplateHandler.java]: I think I get a template error with adding getDisplayName() ? Let me know if that's the case

Comment (thektan) [reply modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/user/facet/template/UserFacetPortletDisplayTemplateHandler.java]: Ah, good catch. That's not a valid method. I'll update it with the correct one.

**Comment by thektan** [comment]
@oliv-yu Updated! ✨

**Comment by thektan** [comment]
ci:test:sf:norebase

**Comment by thektan** [comment]
ci:test:search:norebase

**Comment by thektan** [comment]
ci:test:sf:norebase

**Comment by thektan** [comment]
Submitted to https://github.com/brianchandotcom/liferay-portal/pull/84703

**Pull request closed without merging**